bedrock/products/templates/products/landing.html

```diff
@@ -15,136 +16,214 @@
+          'class': 'mzp-c-picto-image',
+          'width': '64',
+          'height': '64',
+          'loading': 'lazy'
+        }
+      ),
+    ) %}
+      <h3 class="mzp-c-picto-heading"><a href="{{ url('products.vpn.landing') }}" data-cta-type="link" data-cta-text="VPN" data-cta-position="heading">{{ ftl('firefox-products-mozilla-vpn') }}</a></h3>
       <p>{{ ftl('firefox-products-surf-stream-and-get-work-done') }}</p>
-      <p><a class="mzp-c-cta-link" href="{{ url('products.vpn.landing') }}" rel="external noopener" data-cta-type="link" data-cta-text="Get Mozilla VPN">{{ ftl('firefox-products-get-mozilla-vpn') }}</a></p>
-    </div>
-  {% endif %}
-
-    <div class="c-landing-grid-item">
-      <img src="{{ static('img/products/landing/pocket.svg') }}" class="c-landing-grid-img" alt="" width="350" height="247">
-      <h2 class="c-landing-grid-title"><a href="https://getpocket.com/firefox_learnmore/{{ referrals }}">{{ ftl('firefox-products-pocket') }}</a></h2>
+      <p><a class="mzp-c-button mzp-t-product" href="{{ url('products.vpn.landing') + '#pricing' }}" rel="external noopener" data-cta-type="link" data-cta-text="Get VPN">{{ ftl('firefox-products-get-mozilla-vpn') }}</a></p>
+    {% endcall %}
+
+    <!-- MDN Plus -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='img/logos/mdn/mdn-plus-logo.svg',
+        optional_attributes={
+          'class': 'mzp-c-picto-image',
+          'width': '64',
+          'height': '64',
+          'loading': 'lazy'
+        }
+      ),
+    ) %}
+    <h3 class="mzp-c-picto-heading"><a href="https://developer.mozilla.org/{{ referrals }}" data-cta-type="link" data-cta-text="MDN Plus" data-cta-position="heading">{{ ftl('firefox-products-mdn-plus') }}</a></h3>
+      <p>{{ ftl('firefox-products-resources-for-developers') }}</p>
+      <p>
+        <a href="https://developer.mozilla.org/plus/{{ referrals }}" rel="external noopener" class="mzp-c-button mzp-t-product" data-cta-type="link" data-cta-text="Support MDN">{{ ftl('firefox-products-support-mdn') }} {{ icon_external|safe }}</a>
+      </p>
+    {% endcall %}
+
+    <!-- Thunderbird -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='img/logos/thunderbird/logo.svg',
+        optional_attributes={
+          'class': 'mzp-c-picto-image',
+          'width': '64',
+          'height': '64',
+          'loading': 'lazy'
+        }
+      ),
+    ) %}
+      <h3 class="mzp-c-picto-heading"><a href="https://www.thunderbird.net/{{ referrals }}" data-cta-type="link" data-cta-text="Thunderbird" data-cta-position="heading">{{ ftl('firefox-products-thunderbird') }}</a></h3>
+      <p>{{ ftl('firefox-products-access-all') }}</p>
+      <p><a class="mzp-c-button mzp-t-product" href="https://www.thunderbird.net/#free-your-inbox" rel="external noopener" data-cta-type="link" data-cta-text="Thunderbird Download">{{ ftl('firefox-products-download-thunderbird') }} {{ icon_external|safe }}</a></p>
```

Comment:
The "Download" CTA also has no utm params:

```suggestion
      <p><a class="mzp-c-button mzp-t-product" href="https://www.thunderbird.net/thunderbird/all/{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Thunderbird Download">{{ ftl('firefox-products-download-thunderbird') }} {{ icon_external|safe }}</a></p>
```

In this case I'd also suggest instead of linking to `#free-your-inbox` fragment that may change anytime (that provides the same download box as the page intro that is already linked) either link again to the page root, or if you want to provide different link in CTA than in the heading use the /all download page listing all the locales, platforms etc.

Reply:
or maybe better, similar to FF download page autodetecting the platform and starting the download upon opening, there's https://www.thunderbird.net/download/ for that:

```suggestion
      <p><a class="mzp-c-button mzp-t-product" href="https://www.thunderbird.net/download/{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Thunderbird Download">{{ ftl('firefox-products-download-thunderbird') }} {{ icon_external|safe }}</a></p>
```

Reply:
Nice find, I'll use that.

Reply:
Hm, now I'm questioning if triggering a download without warning is the right thing to do. Maybe I'll try to find a download icon so they have some warning.

Reply:
I know what you mean! My first suggestion was actually pointing to `/thunderbird/all` which is analogous to `/firefox/all` here, but there you have to manually pick your platform and such so that felt quite inconvenient. And since Firefox download CTA also starts the download (pointing to `/download/thanks`) immediately, this felt like a good companion — the Thunderbird CTA has the right "Download" wording like FX (compared to other CTAs "Get", "Support", "Check" etc.) — so I assumed these two with "Download" CTA, FX & TB, could be this special category triggering the arch+lang autodetect locale consistently 🤷 

If you feel like distinguishing the action with iconography, there's [download.svg](https://github.com/mozilla/protocol-assets/blob/main/icons/download.svg) in base assets already available if needed.